packages/react-core/src/operations/histogram.d.ts

```diff
@@ -1,9 +1,10 @@
-import { AggregationTypes } from './aggregation/AggregationTypes';
+import { AggregationTypes } from './constants/AggregationTypes';
 import { HistogramFeature } from '../types';
 
-export function histogram(
-  features: [],
-  columnName: string,
-  ticks: number[],
-  operation: AggregationTypes
-): HistogramFeature;
+export function histogram(args: {
+  data: Record<string, unknown>[];
```

Comment:
Rename features and columnName is not necessary, right? it's an unnecessary risk. I can see others variables rename in other files

Reply:
No, it isn't mandatory. But since these fns are internals, I thought it was a good moment to have the same prop names in every file so they're easier to read.